lib/types/applications/applications.go

```diff
@@ -46,6 +49,17 @@ type UtilityName string
 // UtilityFluid is the special utility name for the fluid token itself
 var UtilityFluid UtilityName = "FLUID"
 
+type ApplicationDataAmm struct {
+	FirstToken  ethereum.Address
+	FirstTick   int32
+	SecondToken ethereum.Address
+	SecondTick  int32
+}
+
+type ApplicationData struct {
```

Comment:
Can you explain why we broadened the scope of the applications to include ApplicationData?